stable_baselines3/common/logger.py

```diff
@@ -23,6 +23,28 @@
 DISABLED = 50
 
 
+class Video(object):
+    """
+    Video data class storing the video frames and the frame per seconds
+    """
+
+    def __init__(self, frames: th.Tensor, fps: Union[float, int]):
+        self.frames = frames
+        self.fps = fps
+
+
+class FormatUnsupportedError(NotImplementedError):
```

Comment:
maybe say also which value is not supported by this format

Reply:
You got me there, I was probably a bit lazy on the error message ^^.